src/main/resources/db/migration/V4__unique_constraint_portfolio_items.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE portfolio_items
+ADD CONSTRAINT unique_company_name_user_id UNIQUE (company_name,user_id);
```

Comment:
Please add a new line to the end of file.

Reply:
fixed that